doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -306,7 +306,7 @@ run on both ``cygwin64`` and ``cygwin32``.
 The second form a ``.skipif`` or ``SKIPIF`` file can take is that of a script.
 This form is intended for conditions that require some computation to determine,
 or when the combination of conditions is necessary (i.e. this setting **and**
-this setting are required for the behavior we want to avoid).  The script can be
+this setting is required for the behavior we want to avoid).  The script can be
```

Comment:
It looks like you did a nice close reading of these docs.  This one though was correct as the subject is a pair of settings.  It may help to read it as "setting 1 and setting 2 are required..."

Reply:
Oh. Sorry.